eng/templates/Azure.ResourceManager.Template/.template.config/template.json

```diff
@@ -54,19 +54,43 @@
       "defaultValue": "",
       "replaces": "SwaggerVersionTag"
     },
+    "TypeSpecDirectory": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "description": "TypeSpecDirectory under sdk/providername.",
+      "defaultValue": "TypeSpecDirectory",
+      "replaces": "TypeSpecDirectory"
+    },
+    "SpecOrg": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "description": "the organization name of Azure spec repo.",
+      "defaultValue": "Azure",
+      "replaces": "SpecOrg"
+    },
+    "CommitId": {
```

Comment:
I think this can be updated afterwards, we did not let user input commitId for autorest.md before.

Reply:
we need to setup  essential information at the process of creating new project. So this argument is needed.